### ISSUE TITLE
`DELIMITER ;;` is broken in 3.1.2

### ISSUE THREAD

**Issue by vasek-b** [nextras/migrations#112]
I use `DELIMITER ;;` to change delimiter for trigger definition. It worked on nextras/migrations 3.0.5, but after update to 3.1.2 this construction ends up with an error:

You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'DELIMITER' at line 1

It seems to me that the SQL commands might be split on semicolon which might cause this error.

My settings:
driver: mysql
dbal: nextras
php 7.3
MariaDB 10.3.24

**Comment by hrach** [comment]
Please post the file or minimal reproduction. Thanks.

**Comment by vasek-b** [comment]
I was able to identify the cause - there are two consecutive `DELIMITER` statements without any code between them.

So something like

```sql
DELIMITER ;;
CREATE TRIGGER ...;;
DELIMITER ;

DELIMITER ;;
CREATE TRIGGER ...;;
DELIMITER ;
```

Minimal reproduction that works in 3.0.5 and reports the error on 3.1.2 is

```sql
CREATE TABLE `x` (
	`id` INT
);

DELIMITER ;;
DELIMITER ;
```

**Comment by JanTvrdik** [comment]
@vasek-b Thank you for the reproduction example. It should be fixed in 3.1.3

**Comment by vasek-b** [comment]
Thank you for fixing it so quickly 😉
